src/libs/rabbitmqClient.ts

```
@@ -0,0 +1,197 @@
+    });
+
+    channel = await connection.createConfirmChannel();
+    logger.info('[RabbitMQ] Channel created');
+
+    await channel.assertQueue(ENV.RABBITMQ_FACTCHECK_QUEUE_NAME, {
+      durable: true,
+    });
+    logger.info(
+      `[RabbitMQ] Queue ${ENV.RABBITMQ_FACTCHECK_QUEUE_NAME} asserted & ready`,
+    );
+  } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)